### PR TITLE
Reduce amount of code

### PR DESCRIPTION
**Pull request by GeorgCantor**
Reduced the amount of code using Kotlin Synthetic

### CODE REVIEW

**Comment by nic0lette** [comment]
Hi @GeorgCantor ! Thanks for the PR.

Since Kotlin Android Extensions are deprecated, we don't want to use Kotlin synthetics here. If you were able to [migrate these to use view binding](https://developer.android.com/topic/libraries/view-binding/migration), then I'd be able to accept the PR.

**Comment by GeorgCantor** [comment]
Ok

File: automotive/src/main/java/com/example/android/uamp/automotive/QrCodeSignInFragment.kt [hunk 1]
```diff
@@ -39,34 +34,43 @@ import kotlinx.android.synthetic.main.qr_sign_in.toolbar
  */
 class QrCodeSignInFragment : Fragment(R.layout.qr_sign_in) {
 
+    private var binding: QrSignInBinding? = null
+
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
+        binding = QrSignInBinding.bind(view)
```
Comment (nic0lette) [inline automotive/src/main/java/com/example/android/uamp/automotive/QrCodeSignInFragment.kt:41]: If you make `binding` a local variable then it wouldn't be nullable (so you could get rid of many of the `?`s) and you wouldn't need to clear it in `onDestroy()`

File: automotive/src/main/java/com/example/android/uamp/automotive/QrCodeSignInFragment.kt [hunk 2]
```diff
@@ -39,34 +34,43 @@ import kotlinx.android.synthetic.main.qr_sign_in.toolbar
...
+    }
+
     /**
      * Sets the QR code rendered on this sign in screen.
      *
      * @param code The QR code to display.
      */
     private fun setQrCode(url: String) {
-        Glide.with(this).load(url).into(qr_code)
+        binding?.qrCode?.let { Glide.with(this).load(url).into(it) }
```
Comment (nic0lette) [inline automotive/src/main/java/com/example/android/uamp/automotive/QrCodeSignInFragment.kt:74]: Could you just pass the view in with the url (or, perhaps even better, just move it into `onViewCreated()`)?

File: automotive/src/main/java/com/example/android/uamp/automotive/SettingsActivity.kt [hunk 3]
```diff
@@ -18,19 +18,22 @@ package com.example.android.uamp.automotive
...
-import kotlinx.android.synthetic.main.activity_settings.toolbar
+import com.example.android.uamp.automotive.databinding.ActivitySettingsBinding
 
 /**
  * This class exposes application settings
  * for integration with MediaCenter in Android Automotive.
  */
 class SettingsActivity : AppCompatActivity() {
 
+    private lateinit var binding: ActivitySettingsBinding
```
Comment (nic0lette) [inline automotive/src/main/java/com/example/android/uamp/automotive/SettingsActivity.kt:29]: Does this need to be a member? It looks like everything is set in `onCreate`?

**Review by nic0lette — COMMENTED** [review]
Thanks for the update! Just a few small things but otherwise looks good =D

**Review by nic0lette — APPROVED** [review]
Thanks for the PR!

**Pull request merged**